packages/common-ui-web/test/components/TakePictureButton.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TakePictureButton } from '../../src';
+
+const TAKE_PICTURE_BTN_TEST_ID = 'take-picture-btn';
+const TAKE_PICTURE_OUTER_BTN_TEST_ID = 'take-picture-outer-btn';
+
+describe('TakePictureButton component', () => {
+  it('should take the size prop into account', () => {
+    const size = 60;
+    const borderWidth = 3;
+    const margin = 3;
+    const { unmount } = render(<TakePictureButton disabled={false} size={size} />);
+
+    const outerDiv = screen.getByTestId(TAKE_PICTURE_OUTER_BTN_TEST_ID);
+    const innerDiv = screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID);
+    expect(outerDiv.style.width).toEqual(`${size - (2 * borderWidth)}px`);
+    expect(outerDiv.style.height).toEqual(`${size - (2 * borderWidth)}px`);
+    expect(innerDiv.style.width).toEqual(`${size - (2 * borderWidth) - (2 * margin)}px`);
+    expect(innerDiv.style.height).toEqual(`${size - (2 * borderWidth) - (2 * margin)}px`);
+    unmount();
+  });
+
+  it('should display a take picture button that calls the handle.takePicture method', () => {
+    const handle = { takePicture: jest.fn() };
+    const { unmount } = render(<TakePictureButton disabled={false} onClick={handle?.takePicture} size={60} />);
+
+    fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
+    expect(handle.takePicture).toHaveBeenCalled();
+    unmount();
+  });
```

Comment:
Haha, I think you just copy pasted this test by mistake. Replace this test with :

```typescript jsx
it('should pass the onClick prop to the button', () => {
  const onClick = jest.fn();
  const { unmount } = render(<TakePictureButton onClick={onClick} />);

  fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
  expect(handle.takePicture).toHaveBeenCalled();
  unmount();
});
```